perf/istio-install/istioctl_profiles/istio-non-sds.yaml

```diff
@@ -1,10 +1,9 @@
-apiVersion: install.istio.io/v1alpha2
-kind: IstioControlPlane
+apiVersion: operator.istio.io/v1alpha1
+kind: IstioOperator
 spec:
-  security:
-    components:
-      nodeAgent:
-        enabled: false
+  components:
+    nodeAgent:
+      enabled: false
```

Comment:
is the other one even turning on sds? afaik istioctl does not work with node agent